bHY6yh-d10089/cod-vyg67d_bHY6yh-d10089.xml

```diff
@@ -0,0 +1,308 @@
+           deus si non uigilaueritis ueniam 
+           <lb ed="#O"/>tanquam fur et nescies qua hora ue<lb ed="#O" break="no"/>niam 
+           ad te.</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e423">
+           <!--line bumps down-->
+           <lb ed="#O"/><ref xml:id="bHY6yh-d10089-Rd1e449">Petrus a<!--line bumps up-->postolus epistola. 2. capitulo 3. et fine <!--line bumps back down--> </ref> ¶ <quote xml:id="bHY6yh-d10089-Qd1e427">Adueniet dies domini 
+           sicut fur in quo celi magno im<lb ed="#O" break="no"/>petu
+           transient elementa uero calore 
+           <cb ed="#O" n="b"/>
+           <lb ed="#O"/>soluentur.</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e435">
+           <!--line bumps down-->
+           <lb ed="#O"/><ref xml:id="bHY6yh-d10089-Rd1e463">Paulus <!--line bumps up--> apostolus epistola. 1. ad corinth. <!--line bumps back down--> capitulo 15.</ref> ¶ <quote xml:id="bHY6yh-d10089-Qd1e439">Accepi quoniam christus 
+           <lb ed="#O"/>mortuus est pro peccatis uestis
+           <lb ed="#O"/>scripturas et quia sepultus est et 
+           <lb ed="#O"/>quia resurrexit tertia die secundum scriptu<lb ed="#O" break="no"/>ras. 
+           ¶ Si christus praedicatur quod resurexerit 
+           <lb ed="#O"/>a mortuis quo quadam dicunt in uo<lb ed="#O" break="no"/>bis 
+           quoniam resurrectio mortuorum non est. 
+           <lb ed="#O"/>Si autem christus non resurrexit inanis est 
+           <lb ed="#O"/>ergo praedicatio nostra inanis est ergo et fides 
+           <lb ed="#O"/>nostra. ¶ Surget corpus spirituale. ¶ Ca<lb ed="#O" break="no"/>ro 
+           et sanguis regnum dei possidere 
+           <lb ed="#O"/>non possunt. ¶ Omnes quidem resurge<lb ed="#O" break="no"/>mus 
+           sed non <sic>omnes omnes</sic> in mutabimur 
+           in momento in ictu oculi in nouissima 
+           <lb ed="#O"/>tuba Canet enim tuba et mortui re<lb ed="#O" break="no"/>surgent 
+           primi incorrupti et nos in 
+           <lb ed="#O"/>mutabimur oportet enim corrup<lb ed="#O" break="no"/>tibile 
+           sed induere corruptionem et mor<lb ed="#O" break="no"/>tale 
+           hoc induere inmortalitatem.</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e479">
+           ¶ <ref xml:id="bHY6yh-d10089-Rd1e507">Idem 
+           <lb ed="#O"/>epistola. 2. ad corinth. capitulo 5</ref> <quote xml:id="bHY6yh-d10089-Qd1e481">Omnes uos 
+           <lb ed="#O"/>manifestari oportet ante tribunal
+           <lb ed="#O"/>christi ut referat unusquisque propria 
+           <lb ed="#O"/>corporis prout gessit siue bonum siue 
+           <lb ed="#O"/>malum.</quote>
+        </p>
+        <p xml:id="bHY6yh-d10089-d1e491">
+           <lb ed="#O"/><ref xml:id="bHY6yh-d10089-Rd1e521">Idem epistola. i. ad thymo. capitulo 4.</ref> 
+           ¶ <quote xml:id="bHY6yh-d10089-Qd1e493">In nouissimis temporibus discedent qui<lb ed="#O" break="no"/>dam 
```

Comment:
are we missing a line break here? If no this line should be moved to the previous. 
But I suspect there is a line break missing here.